services/app-web/src/index.tsx

```diff
@@ -136,6 +141,7 @@ if (ldClientId === undefined) {
                     authMode={authMode}
                     apolloClient={apolloClient}
                     s3Client={s3Client}
+                    apiURL={apiURL}
```

Comment:
Why string this variable through as a prop everywhere ? I think its not needed, it is not used across the application logic ( whereas`authMode` is )

I suggest we just access this process.env directly in the `AppAccess.tsx` component and remove the code changes in App/AppBody/AppRoutes entirely.

Reply:
Alright, I can do that. Agreed this is annoying to prop drill so deep. Maybe we could have some kind of config context to read from? Anytime we read env vars out side of index.tsx (or apollo_gql.ts on the back end) it smells a little, making it a little harder to test things cleanly